Yoast/wordpress-seo
# Premium 2159 move stale cornerstone content filter to premium

## Summary

This PR can be summarized in the following changelog entry:

* N/A

## Relevant technical choices:

* Removes the Stale Cornerstone classes and tests from this repo to move them to the Premium repo.

## Test instructions

This PR can be tested by following these steps:

* Make sure you have disabled the Beta flag (to remove the notification)
* Check out this branch
* Enable the Beta flag
* See the notification not being shown anymore
* See the stale content filter not being added to the post-overview

## Quality assurance


Fixes https://github.com/Yoast/wordpress-seo-premium/issues/2159
